### PR TITLE
Update UOS swap infos

### PR DESCRIPTION
**Pull request by envin3**
image
image

### CODE REVIEW

File: src/settings/swap-assets.js [hunk 1]
```diff
@@ -621,6 +621,7 @@ const swapAssets = [
     isPerc20: true,
     chainId: ChainId.UltraMainnet,
     onPnetworkV2: true,
+    formattedName: 'Native UOS',
```
[the remark this answers is not in the document]
Comment (envin3) [reply src/settings/swap-assets.js:624]: This naming is explicitly required.

File: src/components/pages/swap/Swap.jsx [hunk 2]
```diff
@@ -393,6 +393,14 @@ const Swap = ({
                   0x1003d3574ac79303a5fa0951ecb04cc7acba9747) and it is the only official pBTC representation on BSC.
                 </InfoEta>
               ) : null}
+              {to && to.id === 'UOS' ? (
+                <InfoEta>
+                  By bridging to Ethereum you will get native ERC20 UOS tokens passing through the pUOS pToken
+                </InfoEta>
+              ) : null}
+              {to && to.id === PUOS_ON_ULTRA_MAINNET ? (
+                <InfoEta>By bridging to Ultra you will get native UOS tokens passing through the pUOS pToken</InfoEta>
```
Comment (oliviera9) [inline src/components/pages/swap/Swap.jsx:402]:
```suggestion
                <InfoEta>By bridging to Ultra you will get native UOS tokens passing through the pUOS pToken
</InfoEta>
```
Does it change something?

Comment (oliviera9) [reply src/components/pages/swap/Swap.jsx:402]: The above item has the newline. That's why I'm asking

Comment (oliviera9) [reply src/components/pages/swap/Swap.jsx:402]: Automatically managed by prettier/eslint

**Review by oliviera9 — APPROVED** [review]
LGTM

File: src/components/pages/swap/Swap.jsx (hunk 2 above)
Comment (envin3) [reply src/components/pages/swap/Swap.jsx:402]: The reason is prettier rule. Depending on the length of the line it removes or add new lines.
The message is rendered in the same way.

**Pull request merged**